test/e2e/kubernetes/workloads/sgx-test.yaml

```diff
@@ -7,8 +7,7 @@ spec:
     spec:
       containers:
       - name: sgxtest
-        image: oeciteam/sgx-test
-        command: ["/helloworld/host/helloworldhost", "/helloworld/enclave/helloworldenc.signed"]
+        image: oeciteam/sgx-test:1.0
```

Comment:
Shall I assume this image has the equivalent "run something to validate basic SGX functionality and exit 0 if success" ?

Reply:
Yep! It has the same helloworld application, just moved the executing command to the image itself

Reply:
I'm running tests to validate this now, then will merge, thanks!

Btw, do we know why we are only testing Kubernetes v1.15 and v1.16? Does the `sgx-device-plugin` not work w/ v1.17+ for some reason (that would be weird), or do we need to enable tests for those versions of Kubernetes?

Reply:
Where exactly is this test happening? There's a different version of the yaml file for k8s v1.17 onwards because of changes from beta mode

Reply:
We use this config as input to the Jenkinsfile in the root of the project:

https://github.com/Azure/aks-engine/blob/master/test/e2e/test_cluster_configs/sgx.json

Essentially, by building an 18.04-LTS node pool w/ the `Standard_DC2s` VM SKU, we expect that the drivers will be installed and that this `sgx-test.yaml` will be properly scheduled and executed on a node in the cluster.

I'm building a 1.18 cluster now /w a 18.04 + `Standard_DC2s` node pool, I'll report back...

Reply:
The sgx-test.yml mounts /dev/sgx so it essentially doesn't depend on the plugin...